Assets/Scripts/PlayerController.cs

```diff
@@ -41,28 +44,32 @@ public class PlayerController : MonoBehaviour {
     [SerializeField]
     private ContactFilter2D groundContactFilter;
     [SerializeField]
+    private GameObject collectableUI;
+    [SerializeField]
     private Slider overheatSlider;
+    [SerializeField]
+    private AudioClip jumpSound;
+    [SerializeField]
+    private AudioClip deathSound;
+    #endregion
 
+    #region Private Variables
```

Comment:
The technical term for class-wide scope variables is "fields" (I don't know why). https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/fields